modules/core/src/lib/composite-layer.js

```diff
@@ -153,40 +153,40 @@ export default class CompositeLayer extends Layer {
       extensions
     };
 
-    if (sublayerProps) {
-      const overridingSublayerProps = overridingProps && overridingProps[sublayerProps.id];
-      const overridingSublayerTriggers =
-        overridingSublayerProps && overridingSublayerProps.updateTriggers;
-
-      if (overridingSublayerProps) {
-        const propTypes = this.constructor._propTypes;
-        for (const key in overridingSublayerProps) {
-          const propType = propTypes[key];
-          // eslint-disable-next-line
-          if (propType && propType.type === 'accessor') {
-            overridingSublayerProps[key] = this.getSubLayerAccessor(overridingSublayerProps[key]);
-          }
+    const overridingSublayerProps =
+      overridingProps && sublayerProps.id && overridingProps[sublayerProps.id];
```

Comment:
Nit: what is the scenario where `sublayerProps.id` is not specified but `overridingProps[sublayerProps.id]` is?

Reply:
Before this change, `sublayerProps` had no default value, so if  they are not provided, we never access `id` in `sublayerProps` object, but now we have default value, `{}`, so this check basically replaces old one.

Reply:
That is not the question. If `sublayerProps.id` is `undefined` then `overridingProps[undefined]` is not going to fail, so `overridingProps && overridingProps[sublayerProps.id]` should be sufficient.

Reply:
Got it,  updated.